runtime/doc/intro-lua.txt

```diff
@@ -0,0 +1,85 @@
+*into-lua.txt*	Nvim
+
+		     NVIM REFERENCE MANUAL - by Dundar Göc
+
+			Introduction to Lua
+
+
+This chapter will give you the basics of using Lua in Neovim.
+
+                                      Type |gO| to see the table of contents.
+
+==============================================================================
+Introduction
+
+COMMON QUESTIONS
+
+- Is vimscript being deprecated?
+
+NEVER! (idk write something fancy)
+
+- Is my configuration gonna be faster?
+
+Probably not.
+
+- ?
+
+Probably way more, blanking out at the moment.
+
+==============================================================================
+Using Lua from Neovim
+- `:lua` including `:lua =`
+- putting Lua files in rtp  (I think this is covered in lua.txt in some form,
+  although it's really complicated)
```

Comment:
it really isn't: anywhere a `foo.vim` is sourced from, a `foo.lua` also works the same way. No need to count the ways.

Reply:
I mixed it up, I was thinking of lua package path.

Reply:
Ah, yes, that is indeed a bit involved, and not something that needs to be discussed at this level.